src/System.Reflection.Metadata/src/System.Reflection.Metadata.csproj

```diff
@@ -27,6 +27,25 @@
     <Optimize>true</Optimize>
     <ErrorReport>prompt</ErrorReport>
   </PropertyGroup>
+  <PropertyGroup Condition="'$(Configuration)|$(Platform)' == 'Debug_Future|AnyCPU'">
+    <PlatformTarget>AnyCPU</PlatformTarget>
+    <Optimize>false</Optimize>
+    <ErrorReport>prompt</ErrorReport>
+    <CheckForOverflowUnderflow>true</CheckForOverflowUnderflow>
+    <Future>true</Future>
+  </PropertyGroup>
+  <PropertyGroup Condition="'$(Configuration)|$(Platform)' == 'Release_Future|AnyCPU'">
+    <Future>true</Future>
+    <PlatformTarget>AnyCPU</PlatformTarget>
+    <Optimize>true</Optimize>
+    <ErrorReport>prompt</ErrorReport>
+  </PropertyGroup>
+  <PropertyGroup Condition="'$(Future)' == 'true'">
```

Comment:
Interesting. I need to get the v1.2/v1.3 pipeline flowing ASAP. I'll fix up what I have and we can generalize it later. I'll open an issue.